customer-config/checkout-config/che.plulist.v2.json

```diff
@@ -0,0 +1,221 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/extenda/hiiretail-json-schema-registry/master/customer-config/checkout-config/che.plulist.v1.json",
+  "title": "PLU List messages",
+  "type": "array",
+  "items": {
+    "type": "object",
+    "required": [
+      "id",
+      "plu"
+    ],
+    "properties": {
+      "effectiveDate": {
+        "description": "The date when this PLU list should be applied, if present.",
+        "type": "string",
+        "format": "date",
+        "example": "2021-01-01"
```

Comment:
Dates are hard - being we have stores that spans multiple timezone in one contry should we make sure to include it too ?

Reply:
The json schema does not specify the timezone for dates, only date-time. We could add a comment that the date should be considered a UTC date.